code/style.css

```diff
@@ -1,7 +1,264 @@
-/* the + sign is used to select the sibling that comes immediately next to the element you selected.
-in this case it is the div with the answer class next to the div with the active class.
-this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
+/* Style for project 3: business site */
 
-.active + .answer {
-  /* some styling here */
+body, html {
+  height: 100%;
+  font-family: 'Montserrat', sans-serif;
+  font-weight: 300;
+  margin: 0;
+  font-size: 14px;
 }
+
+h1, h2 {
+  font-family: 'flood-std', sans-serif;
+  font-weight: 400;
+  font-style: normal;
+  text-align: center;
+}
+
+/* Hero banner with video */
+#hero {
+  height: 80%;
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+#hero video {
+  width: 100%;
+  height: 80%;
+  position: absolute;
+  top:0;
+  left: 0;
+  z-index: 5;
+  object-fit: cover;
+}
+
+#hero .caption {
+  position: relative;
+  z-index: 20;
+  color: #ffffff;
+  text-align: center;
+}
+
+#hero::after {
+  width: 100%;
+  height: 80%;
+  content:'';
+  position: absolute;
+  z-index: 10;
+  top: 0;
+  left: 0;
+  background: rgba(0,0,0,0.3);
+}
+
+/* Style for page content */
+.page-content {
+  text-align: center;
+  margin-top: 50px;
+}
+
+.photo {
+  text-align: center;
+  padding: 10px;
+  width: 300px;
+}
+
+/* Style for the accordion */
+.accordion {
+  width: 90%;
+  max-width: 1000px;
+  margin: 2rem auto;
+}
+
+.accordion-item {
+  background-color: #ffffff;
+  color: #111111;
+  margin: 1rem 0;
+  border-radius: 0.5rem;
+  box-shadow: 0 2px 5px 0 rgba(0,0,0,0.25);  
```

Comment:
If you wanna continue to explore some box shadow practises, here's something cool to check out:
https://tobiasahlin.com/blog/layered-smooth-box-shadows/